src/components/UI/ROCrateBrowser/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useRef, useState } from 'react';
+import Modal from 'react-modal';
+import './style.css';
+import RoCrateSingleton from 'utils/roCrateSingleton';
+
+Modal.setAppElement('#root');
+
+const modalStyle = {
+  overlay: {
+    zIndex: 2000,
+    position: 'fixed',
+  },
+  content: {
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+    maxHeight: 'calc(100vh - 5em)',
+    overflowY: 'auto',
+    width: '80vw',
+    height: '80vh',
+  },
+};
+
+type ModalProps = {
+  crateUrl?: string;
+  specificCrateFolder?: string;
+  useButtonVariant?: boolean;
```

Comment:
I believe this is unused and can be removed?

Reply:
@MGS-sails I think it is just this (unused) `specificCrateFolder` prop and proptype that remain to be removed. Thanks!